ajax/libs/moonjs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "moonjs",
+  "version": "0.10.0",
+  "filename": "moon.min.js",
+  "description": "A minimal, blazing fast UI library.",
+  "homepage": "http://moonjs.ga/",
+  "keywords": [
+    "moon",
+    "mvvm",
+    "user interface",
+    "UI"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/KingPixil/moongit"
```

Comment:
`    "url": "https://github.com/KingPixil/moongit"` should be
 `    "url": "https://github.com/KingPixil/moon.git"`